src/tests/Login.test.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import { render, fireEvent, screen, waitForElement } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import Login from "../login/Login";
+import { act } from 'react-dom/test-utils';
+
+
+const server = setupServer(
+    rest.post('http://127.0.0.1:5000/login', (req, res, ctx) => {
+        const payload = {
+            username: "MyUsername",
+            password: "12345678",
+        };
+        expect(req.body).toEqual(payload)
+        return res(ctx.json({ 
+            access_token: "fake_access_token",
+            access_expiry: 1594771200
+        }))
+    })
+)
+
+beforeAll(() => server.listen())
+afterEach(() => server.resetHandlers())
+afterAll(() => server.close())
+
+
+
+it('allows the user to login successfully', async () => {
+    render(<Login />)
+
+    fireEvent.change(screen.getByPlaceholderText('Username or Email'), {
+        target: { value: 'MyUsername' },
+    })
+    fireEvent.change(screen.getByPlaceholderText('Password'), {
+        target: { value: '12345678' },
+    })
+    expect(screen.queryByLabelText("response")).not.toBeInTheDocument();
+    
+    act(() => {
+        fireEvent.click(screen.getByRole('button', { name: "Login" }), {
+            target: { value: 'true' },
+        })
+    });
+    expect(screen.queryByLabelText("response")).not.toBeInTheDocument();
```

Comment:
would have been better if the label was errorMessage. So the test would make sense and it would feel login is successful if there is no errorMessage in the document.

Reply:
done